### PR TITLE
22.02 dep fix

### PR DESCRIPTION
**Pull request by BradReesWork**
address cuDF deprecation of "to_array".  
some cleanup

### CODE REVIEW

File: SOURCEBUILD.md [hunk 1]
```diff
@@ -65,11 +67,8 @@ conda deactivate
 
 ```bash
 
-# for CUDA 11.0
-conda env update --name cugraph_dev --file conda/environments/cugraph_dev_cuda11.0.yml
-
-# for CUDA 11.2
-conda env update --name cugraph_dev --file conda/environments/cugraph_dev_cuda11.2.yml
+# Wherer XXX is the CUDA 11 version
```
Comment (rlratzel) [inline SOURCEBUILD.md:70]:
```suggestion
# Where XXX is the CUDA 11 version
```

**Review by rlratzel — APPROVED** [review]
I just noticed a minor typo.

File: .gitignore [hunk 2]
```diff
@@ -82,6 +86,9 @@ python/_external_repositories/
 
 # created by Dask tests
 python/dask-worker-space
+python/cugraph/dask-worker-space
+python/cugraph/cugraph/dask-worker-space
+python/cugraph/cugraph/tests/dask-worker-space
```
Comment (jnke2016) [inline .gitignore:91]: These 3 are helpful because I won't have to exclude them myself before pushing a commit

**Review by jnke2016 — APPROVED** [review]

**Comment by BradReesWork** [comment]
rerun tests

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**